.circleci/codegen_validation/overwrite_with_normalized.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -xe
+
+
+YAML_FILENAME=$1
+
+
+# Allows this script to be invoked from any directory:
+cd $(dirname "$0")
+
+pushd ..
+
```

Comment:
Why two spaces between every block in `.sh` file?

Reply:
removed